src/css/style.css

```diff
@@ -7,6 +7,32 @@
   --button-light-blue: #76A4FF;
   --input-border-blue: #3878F9;
   --gray-op-2: rgb(139, 148, 158, 0.2);
+  --radius: 10px;
+}
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.d-none {
+  display: none !important;
+}
+.mt-2 {
+  margin-top: 0.75rem !important;
+}
+.mt-3 {
+  margin-top: 1rem !important;
+}
+.mt-6 {
+  margin-top: -4rem !important;
+}
+.p-relative {
+  position: relative !important;
+}
+.p-absolute {
+  position: absolute !important;
```

Comment:
Instead of using `!important` (it's a bad practice in CSS, always should exist a way to overwrite other styles without using `!important`), you can get all this "utils classes", and export it to another CSS file, for example: `utils.css`, and import it at the bottom of this file with general styles:

https://github.com/Azordev/search-by-time-github/blob/7eb866a8ad08b9f232eb83f5417898123c09c355/index.html#L3-L13

At the bottom of the line 12.

Then all styles of this CSS document will be replaced with any class of the `utils` CSS document

Reply:
Pro tip here ☝🏻 ✅